backend/middlewares/sessionManager.js

```diff
@@ -27,6 +27,12 @@ module.exports = async (req, res, next) => {
 		return;
 	}
 
+	if ((req.headers.referer && !req.headers.referer.match(`^${config.public_protocol}://${config.cookie_domain}`)) ||
+		(!req.headers.referer && !config.cookie.skipMissingReferer)) {
```

Comment:
We can remove skipMissingReferer.

We need to always accept missing referrer or plugins won't work